storage/persistent_storage/persistent-storage-cinder.adoc

```diff
@@ -13,6 +13,9 @@ shared across the {product-title} cluster.
 Persistent volume claims are specific to a project or namespace and can be
 requested by users.
 
+For {product-title}, automatic migration from OpenStack Cinder in-tree to the Container Storage Interface (CSI) driver is available as a Technology Preview (TP) feature.
+With migration enabled, volumes provisioned using the existing in-tree storage class are automatically migrated to use the OpenStack Cinder CSI storage class. For more information, see xref:../container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration feature].
```

Comment:
**OpenStack Cinder CSI storage class -> OpenStack Cinder CSI driver** will be better.